### ISSUE TITLE
Senior Software Engineer - Ruby Python or Haskell

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#7339]
Senior Software Engineer - Ruby, Python or Haskell<br>
https://ift.tt/2rzEJ44<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0001/8495/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> Berlin and Stuttgart, Germany<br>
<strong>URL:</strong> <a href="https://virtualq.io/en/career/">https://virtualq.io/en/career/</a></p>
<div><b>Important: Please do NOT apply if you don&rsquo;t have any Telecommunication or call center experience!</b></div>
<div><br></div>
<div>- Berlin or Stuttgart (Germany) or Remote -<br>
</div>
<div><br></div>
<div>
<span>Attention Developers!</span>&nbsp;At virtualQ you get the chance to work on an innovative and awarded product with&nbsp;<span>top-notch technology and software management tools</span>. We have a German-American background and like the concept of&nbsp;<span>simplicity, design and flat hierarchy</span>. You have the chance to work in an awesome team atmosphere with lots for individual liberties. Do you want to change the way in which people wait on hold in customer service telephone lines with us? Are you&nbsp;<span>interested in topics like Machine Learning and Voice Automation</span>? Then you should take your chance and join us as a Senior Backend Developer.</div>
<div><b><br></b></div>
<div>
<span><b>IN A NUTSHELL &ndash; WHO WE ARE AND WHAT WE DO</b></span><br>
</div>
<div><span><br></span></div>
<div>virtualQ provides a&nbsp;<span>novel cloud-solution</span>&nbsp;that waits on hold during longer waiting times for callers of Customer Service Centers. Callers can spend their time productively until virtualQ alerts them to call back when it is their turn. Moreover, the&nbsp;<span>smart voice assistant</span>&nbsp;of virtualQ provides answers to frequently asked questions. Thereby, the virtualQ&nbsp;<span>enterprise software</span>&nbsp;creates an unique customer experience, which is currently turning the service industry upside down. On the company&acute;s side the&nbsp;<span>smart virtualQ algorithms</span>&nbsp;enable Customer Service Centers to manage their incoming calls automatically and more efficiently. Well-known companies like&nbsp;<span>Lufthansa Albatros</span>, the German insurances&nbsp;<span>HanseMerkur and Gothar</span>&nbsp;and the Austrian railway provider&nbsp;<span>&Ouml;sterreichische Bundesbahn</span>&nbsp;are already working with our software successfully. Several national and international institutions awarded the innovative cloud-products of virtualQ and the company is backed by an&nbsp;<span>institutional investment fund</span>.</div>
<div><br></div>
<div>
<span><b>WITH THESE ACTIVITIES YOU WILL HAVE A GREAT IMPACT ON OUR BUSINESS</b></span><br>
</div>
<div><span><br></span></div>
<ul>
<li>
<span>Work closely with our CTO</span>&nbsp;and shape our product</li>
<li>Help build the development process of our growing team</li>
<li>
<span>Design Software and Architecture for our Stack</span>, using the latest technologies</li>
<li>Design&nbsp;<span>Voice and Audio Applications / Machine Learning Algorithms&nbsp;</span>and<span>&nbsp;Interfaces</span>, that will shape the future of virtualQ</li>
<li>
<span>Work with a team</span>&nbsp;of great Front and Backend developers</li>
<li>Support&nbsp;<span>DevOps of Amazon AWS hosted Servers, Heroku</span>&nbsp;etc.</li>
<li>Develop&nbsp;<span>Monitoring, Uptime and Security Strategies</span>
</li>
<li><span>Bring in your own ideas and creativity</span></li>
</ul>
<div>&nbsp;<br>
</div>
<div><span><b>WITH THESE SKILLS YOU WILL BE OUR PERFECT CANDIDATE</b></span></div>
<div><br></div>
<div>
<span><b>MUST HAVES</b></span><br>
</div>
<ul>
<li>You love cool tech stacks like&nbsp;<span>Clojure, Haskell, Python</span>
</li>
<li>You know&nbsp;<span>Ruby or Python</span>&nbsp;by heart</li>
<li>You are familiar with<span>&nbsp;AWS, Heroku</span>&nbsp;and the likes</li>
<li>You speak German OR English fluently (native or 2nd language)</li>
<li>You have got a&nbsp;<span>technical education / background</span>
</li>
<li>You do not take things for granted, instead you question the status quo</li>
<li>You have worked in the Telecommunication market before</li>
</ul>
<div><span><br></span></div>
<div><b>NICE TO HAVE</b></div>
<div><b><br></b></div>
<ul>
<li>You have worked with some of the following systems / languages:</li>
<li>Twilio / Nexmo</li>
<li>OpsWorks</li>
<li>Chef or Puppet</li>
<li>Containers / Docker</li>
<li>Linux Servers / Web Servers</li>
<li>Open Source Databases</li>
<li>Machine Learning</li>
<li>Algorithms</li>
</ul>
<div><span><br></span></div>
<div><span><b>WE HAVE GOT A LOT TO OFFER</b></span></div>
<div><br></div>
<ul>
<li>A&nbsp;<span>fair salary</span>&nbsp;&ndash; we might be a startup, but we pay like a company</li>
<li>
<span>Results Only Work Environment</span>&nbsp;&ndash; we do not care when and where you work, we care about your work results. Therefore, we do not count working hours or holidays. Home Office Days are part of our company culture</li>
<li>
<span>Plenty of freedom and opportunities to grow&nbsp;</span>in a young company, which is currently turning the Service Center market upside down</li>
<li>Workshops? Seminars? BarCamps? Tell us what you need to develop your skills and we will make it happen</li>
<li>An&nbsp;<span>awesome, international, open minded team</span>
</li>
</ul>
<div><br></div>
<div>We are an equal opportunities employer and welcome applications from all suitably qualified persons regardless of their race, sex, disability, religion/belief, sexual orientation or age.</div>
<div><b><br></b></div>
<div><span><b>YOUR DREAMJOB MIGHT BE JUST A CLICK AWAY. SO APPLY NOW!</b></span></div>
<div>
<b>Full job description:</b>&nbsp;http://bit.ly/dreamjob_Senior_Software_Engineer</div>
<div>Your contact person:&nbsp;<span>Anika</span>
</div>
<p><strong>To apply:</strong> Please use our online application form: <a href="http://bit.ly/Apply_Now_Senior_Software_Engineer">http://bit.ly/Apply_Now_Senior_Software_Engineer</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
May 11, 2018 at 08:15AM
